src/tools/schema2graph.ts

```diff
@@ -19,7 +19,9 @@ export class SchemaNode {
   // is empty. Otherwise, name is of the form 'ParticleInternal#' and aliases lists the
   // 'Particle_Handle' names that need to be type aliased to it.
   name: string;
+  kotlinName: string;
```

Comment:
Can you add a comment for what this is intended to be? The comment above for 'name' is quite useful.

Reply:
Moved the Kotlin variables to be together and added a comment.